plugin/plugins/mijia/__init__.py

```
@@ -0,0 +1,1204 @@
+        from .mijia_api.core.config import ConfigManager
+        config = ConfigManager()
+        provider = CredentialProvider(config)
+        self.auth_service = AuthService(provider, store)
+
+        # 尝试加载已有凭据
+        credential = await self._load_credential()
+        if credential:
+            try:
+                await self._init_api(credential)
+                self.logger.info("米家插件启动成功，已加载已有凭据")
+            except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)